config.json

```diff
@@ -1,7 +1,7 @@
 {
   "namespace": "phalcon",
   "name": "phalcon",
-  "description": "Web framework delivered as a C-extension for PHP",
```

Comment:
Do not forget to update the documentation and screenshots, for example https://docs.phalcon.io/4.0/en/webserver-setup

![image](https://user-images.githubusercontent.com/1256298/70856198-f7135100-1edf-11ea-93b7-0baf5f688d0b.png)